src/main/java/frc/robot/subsystems/elevator/Elevator.java

```diff
@@ -0,0 +1,63 @@
+package frc.robot.subsystems.elevator;
+
+import edu.wpi.first.units.Distance;
+import edu.wpi.first.units.MutableMeasure;
+import edu.wpi.first.wpilibj.smartdashboard.Mechanism2d;
+import edu.wpi.first.wpilibj.smartdashboard.MechanismLigament2d;
+import edu.wpi.first.wpilibj.smartdashboard.MechanismRoot2d;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.RunCommand;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import org.littletonrobotics.junction.AutoLogOutput;
+import org.littletonrobotics.junction.Logger;
+
+import static frc.robot.subsystems.elevator.ElevatorConstants.MECHANISM_HEIGHT;
+import static frc.robot.subsystems.elevator.ElevatorConstants.MECHANISM_WIDTH;
+
+public class Elevator extends SubsystemBase {
+
+    private static Elevator INSTANCE;
+    private final ElevatorInputsAutoLogged inputs = new ElevatorInputsAutoLogged();
```

Comment:
You create this in the `ElevatorIO`, just take it from there.

Reply:
794888d4abe6b3fac0d2a989e56da7c5ea5d4e9a